packages/angular_devkit/build_angular/src/angular-cli-files/plugins/webpack-rollup-loader.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// Adapted from https://github.com/erikdesjardins/webpack-rollup-loader/blob/master/index.js
```

Comment:
The module resolution logic and base plugin architecture were taken from https://github.com/erikdesjardins/webpack-rollup-loader.

We'll need to either include the original MIT license, or contribute a more general purpose setup back to the original project and use that instead of this modification. I've contacted the author in https://github.com/erikdesjardins/webpack-rollup-loader/issues/25.

Reply:
Resolved by adding an additional license header crediting the original author and license as of used SHA:
```
/**
 * @license
 * @author Erik Desjardins
 * Forked as of SHA 10fb020f997a146725963b202d79290c8798a7a0 from https://github.com/erikdesjardins/webpack-rollup-loader.
 * Licensed under a MIT license.
 * See https://github.com/erikdesjardins/webpack-rollup-loader/blob/10fb020f997a146725963b202d79290c8798a7a0/LICENSE for full license.
 */
```